libraries/SoaringController/MatrixMathExample.pde

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
We don't have pde files any more in the ArduPilot code base.  This will need to be changed.